fe/fe-core/src/main/java/org/apache/doris/plsql/functions/DorisFunctionRegistry.java

```diff
@@ -89,15 +92,57 @@ private String qualified(String name) {
         return (ConnectContext.get().getDatabase() + "." + name).toUpperCase();
     }
 
+    private String getDbName(long catalogId, long dbId) {
+        String dbName = "";
+        CatalogIf catalog = Env.getCurrentEnv().getCatalogMgr().getCatalog(catalogId);
+        if (catalog != null) {
+            DatabaseIf db = catalog.getDbNullable(dbId);
+            if (db != null) {
+                dbName = db.getFullName();
+            }
+        }
+        return dbName;
+    }
+
+    public static boolean like(String str, String expr) {
```

Comment:
refer to `ShowFunctionsStmt`, `ShowEncryptKeysStmt`, `ShowTableCreationStmt`
can rewrite as:
```
 public boolean like(String str) {
        str = str.toLowerCase();
        return str.matches(wild.replace(".", "\\.").replace("?", ".").replace("%", ".*").toLowerCase());
    }
```